OrgInstaller.ps1

```diff
@@ -0,0 +1,414 @@
+                    "inheritedAllow": $PermissionNumber,
+                    "inheritedDeny": 0
+                }
+            }
+        ]
+    }
+"@
+    } | Out-Null
+}
+
+function Install-SimeonDevOpsOrganization {
+    param (
+        $Organization,
+        $ProjectName = "Tenants",
+        $GitHubAccessTokenKeyVaultUrl = "https://installer.vault.azure.net/secrets/$Organization",
+        $ReportingEmailPwUrl = "https://installer.vault.azure.net/secrets/ReportingEmailPw",
+        $SimeonUserToInviteToOrg = "devops@simeoncloud.com",
+        $PipelineNotificationEmail = "pipelinenotifications@simeoncloud.com"
+    )
+
+    Write-Information "Getting required values from Key Vault"
+    Invoke-Command -ScriptBlock {
+        # I know getting the header this way is not correct...
+        $token = Get-SimeonAzureADAccessToken -Resource Vault
+        Write-Information "Getting GitHub access token"
+        $gitHubAccessToken = (Invoke-WithRetry { Invoke-RestMethod -Header @{Authorization = "Bearer $token" } -Uri "$GitHubAccessTokenKeyVaultUrl`?api-version=7.1" -Method Get }).Value
+        if (!$gitHubAccessToken) {
+            throw "Please check that the DevOps organization is correct or contact Simeon Support for a trial."
+        }
+
+        Write-Information "Getting reporting email password"
+        $reportingEmailPw = (Invoke-WithRetry { Invoke-RestMethod -Header @{Authorization = "Bearer $token" } -Uri "$ReportingEmailPwUrl`?api-version=7.1" -Method Get }).Value
+        if (!$reportingEmailPw) {
+            throw "Unable to retrieve reporting email password please contact Simeon Support for assistance."
+        }
+    }
+
+    # Check for org, create if it doesn't exist
+    Write-Information "Validating DevOps organization"
+    Invoke-Command -ScriptBlock {
+        Write-Information "Checking if Organization: $Organization is available"
+        $existingOrg = Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://aex.dev.azure.com/_apis/HostAcquisition/NameAvailability/$Organization" -Method Get }
+        if (!$existingOrg.isAvailable) {
+            $currentUserId = (Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://app.vssps.visualstudio.com/_apis/profile/profiles/me?api-version=5.1" -Method Get }).id
+            $accessToOrgs = (Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://app.vssps.visualstudio.com/_apis/accounts?memberId=$currentUserId&api-version=5.1" -Method Get }).value
+            if ($accessToOrgs.accountName -contains "$Organization") {
+                Write-Information "DevOps Organization: $Organization exists and current user has access"
+            }
+            else {
+                throw "DevOps Organization: $Organization exists but the current user does not have access to $Organization"
+            }
+        }
+        else {
+            Write-Information "Creating DevOps Organization: $Organization"
```

Comment:
I don't suppose there is any way at all to make this a non-AAD connected org in the REST request?